ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestCasesRunner.scala

```diff
@@ -58,13 +58,27 @@ final class LedgerTestCasesRunner(
 
   private def start(test: LedgerTestCase, session: LedgerSession)(implicit
       executionContext: ExecutionContext
+  ): Future[Duration] = {
+    def logAndStart(repetition: Int): Future[Duration] = {
+      logger.info(s"Starting '${test.description}'. Run: $repetition out of ${test.repeated}")
+      startSingle(test, session, repetition)
+    }
+
+    // TODO: make it return all/mean of all durations, not only the last duration
```

Comment:
Is this comment for later? Otherwise, can you remove it if it's already adressed?

Reply:
Actually this is something I couldn't decide if it's worth doing or not.
I've just change this part to return the sum of all durations for repeating test cases. I'm open to alternatives though.